### DISCUSSION TITLE
The new Problem with Evergreen wallet iOS

### DISCUSSION THREAD

**Discussion by Mikael2121** [Chia-Network/chia-blockchain#17183] [Support]
I was send two time Chia from Exchange to my Evergreen wallet. The first time everything worked perfectly, the second time I didn’t receive it although I see a successful transaction in Xchscan:Today can not  See in my wallet even those coins that were on the balance sheet just yesterday   (evegreen mobile ios )

Txn Hash
0xb1b1aad157847fe7546a3ce60f76f3742fa1c245563816cb81a28c925d84e142

Статус
Успешно - Неизрасходовано

Блок
[4,699,511](https://xchscan.com/ru/blocks/4699511)

Отметка времени
8 days ago (Dec-23-2023 22:32:30 PM) +UTC

Значение
5.9404 XCH ($227.4)

Цена XCH
$33.48 / XCH

From:
[xch13sunseryeqqg46ddkcfm9duhpm3spxx4cjjm4j6haqmw49ue4czsy09ndr](https://xchscan.com/ru/address/xch13sunseryeqqg46ddkcfm9duhpm3spxx4cjjm4j6haqmw49ue4czsy09ndr)

To:
[xch1f7qugu6u9lcglfxkzfglldg6n9r3f3jhgwzl90glgzzjlyp6xqwspjx6c7](https://xchscan.com/ru/address/xch1f7qugu6u9lcglfxkzfglldg6n9r3f3jhgwzl90glgzzjlyp6xqwspjx6c7)

Txn Hash
0x405e09049634c2436c77b1c13a479e2057960a3e8822c114d2d866159ded99e8

From: xch13sunseryeqqg46ddkcfm9duhpm3spxx4cjjm4j6haqmw49ue4czsy09ndr

To me: xch1f7qugu6u9lcglfxkzfglldg6n9r3f3jhgwzl90glgzzjlyp6xqwspjx6c7

15.0029 XCH ($567.11)

Please help to find the Problem.

Best regards

**Comment by BrandtH22** [comment]
Hey @Mikael2121 , this sounds like a syncing issue with the wallet but unfortunately the evergreen app was developed and is managed by a third party so you will need to work with their team to determine what might be the cause.

Here is the evergreen discord server (https://discord.gg/sAZTy4DZJ7)

If you decide to use the official desktop chia wallet (https://www.chia.net/downloads/) then we can provide more support by reopening this ticket here or by joining us in our discord server (https://discord.gg/chia).
